File: MinecraftSkinMaker/src/Javafx/Skin.java
package Javafx;

import java.io.File;
//skin object
public class Skin {
    //Location of imgs
    File head;
    File body;
    File lArm;
    File rArm;
    File lLeg;
    File rLeg;

    //consturctor
    Skin (File head, File body, File lArm, File rArm, File lLeg, File rLeg) {
        this.head = head;
        this.body = body;
        this.lArm = lArm;
        this.rArm = rArm;
        this.lLeg = lLeg;
        this.rLeg = rLeg;
    }

    //getters & setters

    public File getHead() {
        return head;
    }

    public void setHead(File head) {
        this.head = head;
    }

    public File getBody() {
        return body;
    }

    public void setBody(File body) {
        this.body = body;
    }

    public File getlArm() {
        return lArm;
    }

    public void setlArm(File lArm) {
        this.lArm = lArm;
    }

    public File getrArm() {
        return rArm;
    }

    public void setrArm(File rArm) {
        this.rArm = rArm;
    }

    public File getlLeg() {
        return lLeg;
    }

    public void setlLeg(File lLeg) {
        this.lLeg = lLeg;
    }

    public File getrLeg() {
        return rLeg;
    }

    public void setrLeg(File rLeg) {
        this.rLeg = rLeg;
    }
}

File: Skin.py
from collections import deque


class Skin:

    # each attribute holds an image
    def __init__(self):
        self.body = None
        self.head = None
        self.shirt = None
        self.pants = None

    # setters
    def set_head(self, a):
        self.head = a

    def set_body(self, a):
        self.body = a

    def set_shirt(self, a):
        self.shirt = a

    def set_pants(self, a):
        self.pants = a

    # getters
    def get_head(self):
        return self.head

    def get_body(self):
        return self.body

    def get_shirt(self):
        return self.shirt

    def get_pants(self):
        return self.pants

